react/features/toolbox/components/web/Toolbox.js

```diff
@@ -894,6 +927,18 @@ class Toolbox extends Component<Props> {
         this._doToggleRecording();
     }
 
+    _onToolbarToggleTranscribing: () => void;
+
+    /**
+     * Dispatched an action to toggle transcribing.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onToolbarToggleTranscribing() {
+        this._doToggleTranscribing();
+    }
```

Comment:
Probably want to add some analytics for this event. While adding analytics in jitsi-meet is pretty inconsistent, I think at least toolbox has analytics for all its buttons.